pipeline/src/transformers/eventDocument.ts

```diff
@@ -126,6 +128,13 @@ export const transformEventDocument = (
       title: asTitle(title),
       caption: primaryImage?.caption && asText(primaryImage.caption),
       series: formatSeriesForQuery(document),
+      times: {
+        startDateTime: documentTimes
+          .map((time) =>
+            time.startDateTime ? new Date(time.startDateTime) : null
+          )
+          .filter((date): date is Date => date !== null),
```

Comment:
General approach is to avoid `null`s in JS - this slightly over the top thought leadery blog post gives some good reasons https://www.split.io/blog/bias-against-null-javascript/;

Instead I would suggest first doing the transform from line 121 for `documentTimes`:
```typescript
const documentTimes = times.map(prismicTimestampToDate)
```
then you can do
```
times: documentTimes
```
on line 121 and then the following here
```suggestion
        startDateTime: documentTimes
          .map(time => time.startDateTime)
          .filter(isNotUndefined),
```

Reply:
Perfect - adjusted!